application/forms/PreferenceForm.php

```diff
@@ -202,6 +203,33 @@ public function createElements(array $formData)
             }
         }
 
+        if (! isset($formData['theme'])
+            || $formData['theme'] === StyleSheet::DEFAULT_THEME
+            || strpos(file_get_contents(StyleSheet::getThemeFile($formData['theme'])), 'prefers-color-scheme') !== false
+            || strpos(file_get_contents(StyleSheet::getThemeFile($formData['theme'])), 'min-height') === false
+        ) {
+            $modes = [
+                'dark'  => $this->translate('Dark'),
+                'light' => $this->translate('Light'),
+                'auto'  => $this->translate('Auto')
+            ];
+            if (strpos(file_get_contents(StyleSheet::getThemeFile($formData['theme'])), 'min-height') === false) {
```

Comment:
You should check this only once.

Reply:
I also would like to see the other options disabled, not invisible. This should work with this element option `'disable' => ['dark', 'light']`.